terraform/modules/front_door/main.tf

```diff
@@ -13,3 +13,12 @@ terraform {
     }
   }
 }
+
+data "aws_caller_identity" "current" {}

```

Comment:
Could have chosen to use something random instead, or let us define a string in each case, but this seemed to make sense and is highly likely to be unique.